business_requirement_deliverable_cost/views/business.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```

Comment:
@victormartinelicocorp My mistake, I forgot tht this s v8.0...
Please replace back to openerp (shame)